source/administration/upgrade.rst

```diff
@@ -6,9 +6,9 @@ In most cases you can upgrade Mattermost Server in a few minutes, but the upgrad
 Which Upgrade Instructions to Use
 ---------------------------------
 
-If you are upgrading from version 3.7.0 or greater
+If you are upgrading from version 3.8.0 or greater
   This guide applies to you. Skip to :ref:`Before you begin <before-you-begin>`.
-If you are upgrading from version 3.0.3 or greater and 3.6.6 or lower
+If you are upgrading from version 3.0.3 or greater and 3.7.5 or lower
```

Comment:
The way it's currently set up, supported versions are in this doc and unsupported versions are in the legacy doc.